docs/topics/time.md

```diff
@@ -0,0 +1,416 @@
+[//]: # (title: Time)
+
+The Kotlin standard library gives you the tools you need to be able to calculate and measure time in different units.
+Being able to measure time accurately is important for activities like:
+  * Managing threads or processes
+  * Collecting statistics
+  * Detecting timeouts
+  * Debugging
+
+By default, time is measured using a monotonic time source but other time sources can be configured.
+For more information, see [Create time source](#create-time-source).
+
+## Durations
+
+To represent an amount of time, the standard library has the [`Duration`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/)
+class. A `Duration` can be expressed in the following units from the [`DurationUnit`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration-unit/)
+enum class:
+  * `NANOSECONDS`
+  * `MICROSECONDS`
+  * `MILLISECONDS`
+  * `SECONDS`
+  * `MINUTES`
+  * `HOURS`
+  * `DAYS`
+
+`Durations` can be positive, negative, zero, positive infinity, or negative infinity.
+
+### Create duration
+
+To create a `Duration`, use the [extension properties](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/#companion-object-properties) 
+available for `Int`, `Long`, and `Double` types: `nanoseconds`, `microseconds`, `milliseconds`, `seconds`, `minutes`, 
+`hours`, and `days`.
+
+> Days refer to periods of 24 hours. They are not calendar days.
+> 
+{type = "tip"}
+
+For example:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.nanoseconds
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Duration.Companion.days
+
+fun main() {
+    //sampleStart
+    val fiveHundredMilliseconds: Duration = 500.milliseconds
+    val zeroSeconds: Duration = 0.seconds
+    val tenMinutes: Duration = 10.minutes
+    val negativeNanosecond: Duration = (-1).nanoseconds
+    val infiniteDays: Duration = Double.POSITIVE_INFINITY.days
+    val negativeInfiniteDays: Duration = Double.NEGATIVE_INFINITY.days
+
+    println(fiveHundredMilliseconds) // 500ms
+    println(zeroSeconds)             // 0s
+    println(tenMinutes)              // 10m
+    println(negativeNanosecond)      // -1ns
+    println(infiniteDays)            // Infinity
+    println(negativeInfiniteDays)    // -Infinity
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-time-create-duration"}
+
+You can also perform basic arithmetic with `Durations`:
```

Comment:
Because the formatting looks a bit ugly as `Duration`s, I decided to change it to `Duration` objects instead. I've applied this change for all the other similar cases of this 😄